packages/db/src/scripts/seed.ts

```
@@ -0,0 +1,145 @@
+    console.log(`📊 Summary:`);
+    console.log(`   - Drivers: ${drivers.length}`);
+    console.log(`   - Laps: ${laps.length}`);
+    console.log(`   - Telemetry Packets: ${packets.length}`);
+    console.log("=".repeat(50));
+    console.log("\n💡 You can now query the database or start the server!");
+    console.log("   - View drivers: SELECT * FROM driver;");
+    console.log("   - View laps: SELECT * FROM lap LIMIT 10;");
+    console.log("   - View telemetry: SELECT * FROM telemetry_packet LIMIT 10;\n");
+
+  } catch (error) {
+    console.error("\n❌ Error seeding database:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)